py-polars/polars/internals/expr.py

```diff
@@ -1108,7 +1396,29 @@ def dot(self, other: Expr | str) -> Expr:
 
     def mode(self) -> Expr:
         """
-        Compute the most occurring value(s). Can return multiple Values
+        Compute the most occurring value(s).
+        Can return multiple Values.
+
+        Examples
+        --------
+
+        >>> df = pl.DataFrame(
+        ...     {
+        ...         "a": [1, 1, 2, 3],
+        ...         "b": [1, 1, 2, 2],
+        ...     }
+        ... )
+        >>> df.select(pl.col("*").mode())
```

Comment:
Can we use pl.all()? That is preferred over wildcard.

Reply:
Done